src/styles/style.css

```diff
@@ -0,0 +1,215 @@
+body {
+  display: flex;
+  height: 100vh;
+  justify-content: center;
+  align-items: center;
+  border: 2px solid red;
+}
+
+.day {
+  box-sizing: border-box;
+  width: 100px;
+  height: 100px;
+  background-color: #eee;
+  border: 1px solid black;
```

Comment:
Avoid using multiple border declarations with different widths and colors for the same element. It's better to define a single border style to maintain consistency and avoid conflicts.